cameraview/src/main/java/com/otaliastudios/cameraview/preview/CameraPreview.java

```diff
@@ -139,7 +139,6 @@ public final T getView() {
      * or a SurfaceTexture).
      * @return the surface object
      */
-    @NonNull
```

Comment:
Sometimes, when you leave the screen very fast before the camera is initialized, this method returns null.